src/main/docs/guide/spring/springWithGradle.adoc

```diff
@@ -0,0 +1,25 @@
+To use micronaut-openapi with spring in gradle add this code to you `build.gradle`:
+
+.build.gradle
+[source,groovy]
+----
+dependencies {
+
+    // add to annotationProcessor and compileOnly blocks next libraries:
+
+    annotationProcessor platform("io.micronaut.platform:micronaut-platform:$micronautVersion")
```

Comment:
I don't think we should promote applying the Micronaut platform BOM, they should only apply the core BOM, the spring BOM and the OpenAPI BOM

The platform BOM may conflict

Reply:
Yes, the versions may not be exactly the same, but personally I use it exactly like this and so far everything has worked fine. It's just that if you don't do it like this, you'll have to set your own version for each library or connect as many as 4 additional bomb files - core, openapi, serde and micronout-spring.

It seems to me that it's quite normal for the first version. If users have real problems, then the example can be improved.

Reply:
so the issue is it might work for you but it is likely not to align the Spring Platform BOM which will create issues for others depending on the dependencies specified. If it is purely documentation then sure maybe it is fine, but be aware this is not going to work at scale

Reply:
ok, fixed